tests/test_add_sections.py

```diff
@@ -1,11 +1,12 @@
 import re
 from pathlib import Path
 
-from sphinx_testing import with_app
+import pytest
 
 
-@with_app(buildername="html", srcdir="doc_test/add_sections")
-def test_section_is_usable_in_filters(app, status, warning):
+@pytest.mark.parametrize("create_app", [{"buildername": "html", "srcdir": "doc_test/add_sections"}], indirect=True)
+def test_section_is_usable_in_filters(create_app):
+    app = create_app
```

Comment:
2 small thinks:
1) Can we rename `create_app` to app, test_app, or sphinx_app?
The app is already created and usable when it arrives in the "tests".

2) `app = create_app` is not needed, we can directly use `create_app.build()...`